docker/README.md

```diff
@@ -19,8 +19,9 @@ mvn clean package -P docker,Tomcat -DskipTests
 
 ## Running a test image
 
-First make sure that you can access the auxiliary images. This can be done by building them yourself, or by using the
-prebuild images in the private Docker registry `private.docker.nexus.frankframework.org`. The private repository
+First make sure that you can access the auxiliary images. This can be done by:
+- building them yourself, by checking out https://github.com/frankframework/ci-images/ and proceed inside directory filesystems/samba2
```

Comment:
Het is natuurlijk niet alleen voor het samba2 image 😄

Reply:
Euh ja, crap...dat had ik bij de unittest willen zetten.. Ik pas het aan. Thanks 😉